wbooks/src/redux/book/reducer.ts

```diff
@@ -13,8 +13,9 @@ const initialState: BookState = {
 
 function bookReducer(state = initialState, action: Action): BookState {
   switch (action.type) {
-    case actions.GET_BOOKS:
-      return { ...state, books: BOOKS_MOCK };
+    case actions.GET_BOOKS_SUCCESS:
+      return { ...state, books: action.payload as Book[] };
+    case actions.GET_BOOKS_FAILURE:
```

Comment:
Missing loading and error variables in the initial state and in the reducer

Reply:
Done!